### PR TITLE
build for rhel7 and rhel8 userspaces

### PR DESCRIPTION
**Pull request by lsm5**
Dockerfile template from Doug Smith <dosmith@redhat.com>

Signed-off-by: Lokesh Mandvekar <lsm5@fedoraproject.org>

@dougbtv @mrunalp PTAL.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,21 +1,31 @@
 # This dockerfile is specific to building Multus for OpenShift
-FROM openshift/origin-release:golang-1.10 as builder
-
-# Add everything
+FROM registry.access.redhat.com/ubi8 as rhel8
 ADD . /usr/src/plugins
-
 WORKDIR /usr/src/plugins
+RUN dnf install -y golang
```
Comment (squeed) [inline Dockerfile]: We prefer to pin golang releases - can you find a way to do this?

Comment (lsm5) [reply Dockerfile]: So, IIUC, @jupierce and @sosiouxme will give us a rhel8 image which will have golang installed already. I'll revise this PR once that's done.

**Comment by lsm5** [comment]
@jupierce @dougbtv @squeed PR updated to use new images, PTAL..

**Comment by lsm5** [comment]
I see this got merged, so I'm gonna close it.

**Pull request closed without merging**